package.json

```diff
@@ -1,22 +1,36 @@
 {
   "name": "vex-js",
-  "version": "2.3.4",
-  "description": "Beautiful, functional, dialogs in Javascript",
-  "author": "Adam Schwartz <aschwartz@hubspot.com>",
+  "version": "3.0.0",
+  "description": "Beautiful, functional dialogs in vanilla JavaScript",
+  "contributors": [
+    "Adam Schwartz <aschwartz@hubspot.com>",
+    "Brendan Batliner <brendan@omnipointment.com>"
+  ],
   "license": "MIT",
-  "main": "js/vex.js",
+  "main": "dist/vex.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/HubSpot/vex.git"
   },
+  "dependencies": {
```

Comment:
Would it be possible to pin all of the dependency versions? Ideally, we should keep dependencies up-to-date, but if they do happen to change, it can make reproducing issues a bit more difficult since the builds may be inconsistent.